pronotepy/clients.py

```diff
@@ -302,7 +315,7 @@ def lessons(self, date_from: datetime.date, date_to: datetime.date = None) -> Li
                 output.append(dataClasses.Lesson(self, lesson))
 
         # since we only have week precision, we need to make it more precise on our own
-        return [lesson for lesson in output if date_from <= lesson.start.date() <= date_to]
+        return [lesson for lesson in output if date_from <= lesson.start <= date_to]
```

Comment:
The list ends up empty in unit tests. Not sure why but something is broken.

Reply:
done